### PR TITLE
Get rid of 'async' in the qunit tests (close #24)

### LINKED ISSUE #24: Get rid of 'async' in the qunit tests

### PR DESCRIPTION
**Pull request by miherlosev**
\cc @inikulin 

1)remove 'async' from tests
2)remove 'async' from production code (transport.js)
3)move tests for Transport from TestCafe to Hammerhead
4)turn on babel runtime on client side (for cross-browser Promise support)
5)report bug for webmake - https://github.com/zloirock/core-js/issues/98

### CODE REVIEW

**Comment by inikulin** [comment]
\r-

File: test/client/fixtures/sandboxes/event/focus-blur-change.js [hunk 1]
```diff
@@ -207,58 +196,56 @@ function testChanging (numberOfHandlers, testCanceled, testCallback) {
     var input2ChangedCount = 0;
 
     var assertChanging = function () {
-        strictEqual(input1ChangeHandlersExecutedAmount, input1ChangedCount, 'input1ChangeHandlersExecutedAmount checked');
-        strictEqual(input2ChangeHandlersExecutedAmount, input2ChangedCount, 'input2ChangeHandlersExecutedAmount checked');
+        return new Promise(function (resolve) {
+            strictEqual(input1ChangeHandlersExecutedAmount, input1ChangedCount, 'input1ChangeHandlersExecutedAmount checked');
```
Comment (inikulin) [inline test/client/fixtures/sandboxes/event/focus-blur-change.js]: So, asserts are sync actually? Then we don't need to promisify them. We need only async funcs to return promise.

**Comment by miherlosev** [comment]
Found issue for webmake - https://github.com/medikoo/modules-webmake/issues/53.
Waiting for @medikoo answer.

**Comment by miherlosev** [comment]
Create issue for promise - https://github.com/then/promise/issues/104

**Comment by miherlosev** [comment]
use es6-promise for client part

**Comment by miherlosev** [comment]
ping @inikulin

**Comment by inikulin** [comment]
LGTM, but we need yet another reviewer. \cc @AlexanderMoskovkin

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Pull request merged**